src/components/core/DpEditor/DpEditor.vue

```diff
@@ -373,6 +375,12 @@ export default {
   mixins: [prefixClassMixin],
 
   props: {
+    basicAuth: {
```

Comment:
Can you add a short docblock here, too?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/567/commits/55c3ae63b443d85b1ea29de3ed94bf27d8193438

Reply:
I'd suggest to change the description to "The Tus endpoint requires basicAuth to be added to the file header.".

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/567/commits/c3b38316ab1710741b6f7d284e3a54b75865b3fa